packages/git.rb

```diff
@@ -83,4 +83,11 @@ def self.check
       abort 'git-remote-https is broken'.lightred
     end
   end
+
+  def self.postinstall
+    puts 'Running git garbage collection...'.lightblue
+    Dir.chdir("#{CREW_PREFIX}/lib/crew") do
```

Comment:
Why the need to cd into this directory?

Reply:
Because `git` operates inside the repo it is running in...

Reply:
I might be mistaken but isn't this outside the repo?

Reply:
That is the repo:
```
chronos@rpi4b-armv7l /usr/local/lib/crew/packages $ cd ..
chronos@rpi4b-armv7l /usr/local/lib/crew $ cd .git
chronos@rpi4b-armv7l /usr/local/lib/crew/.git $ ls
branches  config  config.worktree  description  FETCH_HEAD  HEAD  hooks  index  info  logs  objects  ORIG_HEAD  packed-refs  refs  shallow
chronos@rpi4b-armv7l /usr/local/lib/crew/.git 
```